src/lib/core/backtrace.c

```diff
@@ -98,6 +98,8 @@ collect_current_stack(struct backtrace *bt, void *stack)
 /*
  * Restore target fiber context (if needed) and call `collect_current_stack`
  * over it.
+ * It is guaranteed that if NULL passed as fiber argument, fiber
+ * module will not be used.
```

Comment:
Nit: please also update the comment to the function declaration in the header.

Reply:
Done